horace_core/sqw/@sqw/disp2sqw_eval.m

```diff
@@ -78,25 +78,30 @@
 all_bins=false;
 ave_pix=false;
 if exist('opt','var')  % no option given
-    if ischar(opt) && ~isempty(strmatch(lower(opt),'all'))    % option 'all' given
+    if ischar(opt) && startsWith(opt,'-all', 'IgnoreCase', true)
```

Comment:
This would detect the option:
`-allTheseOptionsAreStupid`, do we want this behaviour?

Reply:
Would strcmpi be better?

Reply:
it's `startswith` , but the other way i.e. `-a`, `-al` and `-all` should match, but not `-allTheseOptionsAreStupid`